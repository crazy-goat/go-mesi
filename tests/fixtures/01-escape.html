<!DOCTYPE html>
<html>
<head>
    <title>Test ESI</title>
</head>
<body>
<!--esi comment with spacer -->here
[<!--esi comment without space-->] here
<!--esi
Multi line, first line with new line-->
<!--esi
Multi line, first line: new line, last line: new line
-->
<!--esi Multi line, last line: new line
-->
<!--esi some html <b>tag</b> -->
</body>
</html>