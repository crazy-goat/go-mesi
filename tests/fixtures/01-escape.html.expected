<!DOCTYPE html>
<html>
<head>
    <title>Test ESI</title>
</head>
<body>
comment with spacer here
[comment without space] here
Multi line, first line with new line
Multi line, first line: new line, last line: new line

Multi line, last line: new line

some html <b>tag</b> 
</body>
</html>